btcwallet/btcwallet_test.go

```diff
@@ -0,0 +1,34 @@
+package btcwallet
+
+import (
+	"sync"
+	"testing"
+
+	_ "golang.org/x/tools/go/expect"
+)
```

Comment:
Not sure what is this for?

Reply:
Thanks, that's where so much garbage in go.sum comes from